lib/services/models/AuthenticationProof.dart

```diff
@@ -1,12 +1,11 @@
 import 'dart:async';
 import 'dart:convert';
-import 'package:flutter_background_geolocation/flutter_background_geolocation.dart';
 import 'package:http/http.dart' as http;
 import 'package:shared_preferences/shared_preferences.dart';
 
 import 'package:bien_aca_quarantine/services/models/User.dart';
 
-final serverUrl = 'http://localhost:3000';
+final serverUrl = 'http://10.0.2.2:3000';
```

Comment:
Change back this to server URL

Reply:
This actually was on localhost, not on server URL. I'll change all local URL back to heroku server